tests/aiohttp/test_aiohttp_1.py

```diff
@@ -55,11 +55,11 @@ async def test_run():
 
   #Make test call
   async with aiohttp.ClientSession() as session:
-    async with session.post('https://petstore.swagger.io/v2/pet', data='{"id":0,"category":{"id":0,"name":"doggie"},"name":"doggie","photoUrls":["http://example.co"],"tags":[{"id":0,"name":"doggie"}],"status":"available"}', headers={ "Accept":"application/json", "Content-Type": "application/json", 'tester1': 'tester1', 'tester2':'tester2' }) as response:
+    async with session.post('http://localhost:8000/route1', data='{ "a":"b", "c": "d" }', headers={ 'tester1': 'tester1', 'tester2':'tester2' }) as response:
```

Comment:
I'd really prefer to use meaningful values for the tests, this way it is easier to catch dummy issues.